scipy/spatial/ckdtree/src/build.cxx

```diff
@@ -13,6 +13,10 @@
 #include <stdexcept>
 #include <ios>
 
+#if defined(__cplusplus)
```

Comment:
This is always a C++ file and I don't see unqualified usage in this file, but if it's necessary then I suspect Cython does something odd here. Happy to not investigate, since it's harmless to merge as is.